### PR TITLE
feat: added vista exp, new bio

### PR DESCRIPTION
**Pull request by rak3rman**
Updated my web presence to reflect my new job at Vista Power as Lead SWE! Included: new bio, new hero tagline, new summarize data.

### CODE REVIEW

File: nuxt/content/index.md [hunk 1]
```diff
@@ -7,55 +7,54 @@ head.image: "https://imagedelivery.net/5zM6Rdl2uV8Hmr9WxRh20g/fa35b458-67ec-4711
...
-manufacturing, consulting, university administration, political campaigns, enterprise security systems, and athletic
-special events. Just this past summer, I was a Software Engineering Intern on the
+I’ve worked across industries including commercial clean energy, spacecraft manufacturing, consulting, university administration, political campaigns, enterprise security systems, film + television production, and athletic special events. Through each experience, I’ve proven I can adapt quickly to new challenges, be industry agnostic, and deliver exceptional software.
+
+Nowadays, I lead all software engineering efforts at [Vista Power](https://vistapower.com). We deliver clean energy as a service for commercial, industrial, and municipal facilities at low or no upfront cost, powered by safe, modular, and cost-effective solar power and battery energy storage. We're moving incredibly fast and are dangerously talent-dense (entire leadership team is ex-SpaceX).
+
+In 2024, I graduated from the [University of Illinois Chicago](https://uic.edu) with a degree in Computer Science and Business. During my time at UIC, I project-managed and worked directly with
```
[the remark this answers is not in the document]
Comment (rak3rman) [reply nuxt/content/index.md:37]: Incorrect, not applying.

File: nuxt/components/content/HomeExperience.vue [hunk 2]
```diff
@@ -118,7 +118,7 @@
         >
           <NuxtImg
             v-for="url in ExperienceLogoWhiteSet"
-            class="max-h-10 w-full object-contain object-left"
+            class="max-h-10 w-[9rem] md:w-[11.5rem] object-contain object-left"
```
Comment (rak3rman) [inline nuxt/components/content/HomeExperience.vue:121]: I believe this is causing a problem with the viewport padding, right side. See screenshot 
image

Comment (rak3rman) [reply nuxt/components/content/HomeExperience.vue:121]: New headshot image scaling was the issue. Pushed a patch. Marking as resolved.

**Comment by seyfal** [comment]
AYYO This is hype! Good Job Radison 🚀

**Pull request merged**